news-bundle/contao/dca/tl_module.php

```diff
@@ -105,6 +105,14 @@
 	'sql'                     => array('type' => 'boolean', 'default' => false)
 );
 
+$GLOBALS['TL_DCA']['tl_module']['fields']['news_keepCanonical'] = array
+(
+	'inputType'               => 'checkbox',
+	'default'                 => true,
+	'eval'                    => array('tl_class'=>'w50'),
+	'sql'                     => array('type' => 'boolean', 'default' => false)
```

Comment:
False or true now?

Reply:
Default is false (generate the archive URL) and the migration keeps the current state.

Reply:
Yeah but why is `default` different from the SQL default?

Reply:
Fixed in 6221554826533b242d44f33091cd6f8288645ed3